src/config.ts

```diff
@@ -219,4 +220,5 @@ export default {
   VORTEX_API_BASE,
   VORTEX_APP_ID,
   VORTEX_TOKEN,
+  WEAVIATE_API_BASE,
```

Comment:
quick note on the URL. It may expect the internal address: `http://weaviate-internal.default.svc.cluster.local:8080`